way_to_home/user/models.py

```diff
@@ -1 +1,55 @@
 """This module implements class that represents the user entity."""
+
+from django.contrib.auth.base_user import AbstractBaseUser
+from django.db import models, IntegrityError
+
+
+class User(AbstractBaseUser):
+    """Model for User entity."""
+
+    email = models.EmailField(max_length=64, unique=True)
+    password = models.CharField(max_length=128)
+    google_token = models.CharField(blank=True, max_length=255)
+    phone_number = models.CharField(blank=True, max_length=15)
+
+    def __str__(self):
+        """Method that returns route instance as string."""
+
+        return f"{self.email}"
+
+    def to_dict(self):
+        """Method that returns dict with object's attributes."""
+
+        return {
+            'id': self.id,
+            'email': self.email,
+            'phone_number': self.phone_number
+        }
+
+    @classmethod
+    def create(cls, email=None, password=None, google_token=None, phone_number=None):  # pylint: disable=arguments-differ
```

Comment:
```suggestion
    def create(cls, email=None, password=None, google_token=None, phone_number=None):  # pylint: disable=arguments-differ
```
```suggestion
    def create(cls, email=None, password=None, google_token=None, phone_number=None):  # pylint: disable=arguments-differ
```
I think fields with `blank=True` must have default value empty string instead of `None`

Reply:
Changed